### ISSUE TITLE
fix: errors  when starting the game

### ISSUE THREAD

**Issue by zwindler** [zwindler/gocastle#5]
```
2023/07/17 21:37:06 Fyne error:  Attempt to access current Fyne app when none is started
2023/07/17 21:37:06   At: /home/zwindler/go/pkg/mod/fyne.io/fyne/v2@v2.3.5/app.go:104
2023/07/17 21:37:06 Fyne error:  Attempt to access current Fyne app when none is started
2023/07/17 21:37:06   At: /home/zwindler/go/pkg/mod/fyne.io/fyne/v2@v2.3.5/app.go:104
2023/07/17 21:37:06 Fyne error:  Attempt to access current Fyne app when none is started
2023/07/17 21:37:06   At: /home/zwindler/go/pkg/mod/fyne.io/fyne/v2@v2.3.5/app.go:104
2023/07/17 21:37:06 Fyne error:  Attempt to access current Fyne app when none is started
2023/07/17 21:37:06   At: /home/zwindler/go/pkg/mod/fyne.io/fyne/v2@v2.3.5/app.go:104
```

**Comment by zwindler** [comment]
Partly fixed by c4bf4d834bbff2f9dfecf76b05ae66eb2c419fcf

**Comment by zwindler** [comment]
For some reason, fyne 2.4.0 seem to fix this. Let's merge #63 first and see if it was a fyne bug?

**Comment by zwindler** [comment]
fixed by #63
